src/MenuMixin.js

```diff
@@ -225,11 +226,12 @@ const MenuMixin = {
       props.className,
       `${props.prefixCls}-${props.mode}`,
     );
+
     const domProps = {
       className,
-      role: 'menu',
-      'aria-activedescendant': '',
```

Comment:
Why?

Reply:
@fatelei Why do you remove this line?

Reply:
aria-activedescendant="" is an invalid value